peer-monitoring-service/client/src/peer_states/latency_info.rs

```diff
@@ -109,6 +109,14 @@ impl LatencyInfoState {
         }
     }
 
+    /// Returns the latest latency ping in seconds. If no latency
+    /// pings have been recorded, None is returned.
+    pub fn get_latest_latency_ping_secs(&self) -> Option<f64> {
+        self.recorded_latency_ping_durations_secs
+            .last_key_value()
+            .map(|(&_key, &value)| value)
```

Comment:
nit: this might look neater as `.map(|_, value| *value)`?

Reply:
fixed, thanks!